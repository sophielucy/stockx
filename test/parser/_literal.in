5.5556;
42;
"hello world";
true;
false;
