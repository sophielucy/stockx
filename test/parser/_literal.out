FloatLiteral(5.5556)
IntLiteral(42)
StringLiteral(hello world)
BoolLiteral(true)
BoolLiteral(false)
